R/check.R

```diff
@@ -32,14 +32,16 @@ check_config = function() {
   config = load_config()
   f = find_config()
   msg_init('Checking ', f)
-  open_file(f)
+  okay = TRUE
```

Comment:
We could set here the option to configure the default value by a user. For example:
```suggestion
  okay = get_option("blogdown.check.open_file", TRUE)
```

It adds an option and not sure it is worth it really to have this configured.